webcompat/form.py

```diff
@@ -147,6 +148,12 @@ def domain_name(url):
     # testing if it's an http URL
     if url.startswith(SCHEMES):
         domain = urlparse.urlparse(url).netloc
+        if domain.lower().startswith('www.'):
+            domain = domain.split('www.')[1]
+        port_number = re.search(r':\d+$', domain)
+        if port_number is not None:
+            if domain.endswith(port_number.group()):
```

Comment:
because of the previous if on `port_number`, this next `if` will always be true, so it's useless. You are basically testing this:

```
>>> domain
'www.org:5000'
>>> port_number.group()
':5000'
>>> domain.endswith(port_number.group())
True
>>> 'www.org:5000'.endswith(':5000')
True
```